website/docs/reference/datasources/querying-oracle.md

```diff
@@ -1,219 +1,101 @@
 ---
 sidebar_position: 11
+description: Connect Appsmith to an Oracle database and create queries.
 ---
 
 # Oracle
 
-This page describes how to connect your application to your Oracle databases and use queries to manage its content.
+This page gives information to connect Appsmith to an Oracle database and to query its data in your applications.
 
-## Configuration
+## Connection parameters
```

Comment:
If you check PostgreSQL, there is a missing heading above this one and also the callout should be moved here too.